docs/howto.rst

```diff
@@ -126,101 +125,146 @@ This is outputted to the terminal:
 
-Therefore, we need to change the ``-thr`` input until ``phys2bids`` finds the correct number of timepoints. Looking at the tutorial_file_trigger_time.png file can help determine what threshold is most appropriate. For this tutorial file, a threshold of 0.735 finds the right number of time points.
+Let's go through an example where the number of timepoints automatically found is not correct. For that, will we use tutorial_file_v2.txt (in the same location as tutorial_file.txt):
 
 .. code-block:: shell
 
-    phys2bids -in tutorial_file -chtrig 1 -outdir /home/arthurdent/physio -ntp 158 -tr 1.2 -thr 0.735
+    phys2bids -in tutorial_file_v2.txt -indir /home/arthurdent/git/phys2bids/phys2bids/tests/data/ -chtrig 1 -ntp 158 -tr 1.2 -outdir /home/arthurdent/physio_v2
 
-    INFO:phys2bids.phys2bids:Currently running phys2bids version v1.3.0-beta+152.g1f98d16.dirty
-    INFO:phys2bids.phys2bids:Input file is tutorial_file.txt
-    INFO:phys2bids.utils:File extension is .txt
-    WARNING:phys2bids.utils:If both acq and txt files exist in the path, acq will be selected.
-    INFO:phys2bids.phys2bids:Reading the file ./tutorial_file.txt
-    INFO:phys2bids.interfaces.txt:phys2bids detected that your file is in Labchart format
-    INFO:phys2bids.phys2bids:Reading infos
-    INFO:phys2bids.physio_obj:
+The output:
+
+.. code-block:: shell
     ------------------------------------------------
-    File tutorial_file.txt contains:
+    File tutorial_file_v2.txt contains:
     01. Trigger; sampled at 1000.0 Hz
     02. CO2; sampled at 1000.0 Hz
     03. O2; sampled at 1000.0 Hz
     04. Pulse; sampled at 1000.0 Hz
     ------------------------------------------------
 
-    INFO:phys2bids.viz:saving channel plot to tutorial_file.png
     INFO:phys2bids.physio_obj:Counting trigger points
+    INFO:phys2bids.physio_obj:The number of timepoints according to the std_thr method is 157. The computed threshold is 1.1506977594943586
     INFO:phys2bids.physio_obj:Checking number of timepoints
-    INFO:phys2bids.physio_obj:Found just the right amount of timepoints!
+    WARNING:phys2bids.physio_obj:Found 1 timepoints less than expected!
+    WARNING:phys2bids.physio_obj:Correcting time offset, assuming missing timepoints are at the beginning (try again with a more conservative thr)
     INFO:phys2bids.phys2bids:Plot trigger
     INFO:phys2bids.phys2bids:Preparing 1 output files.
     INFO:phys2bids.phys2bids:Exporting files for freq 1000.0
     INFO:phys2bids.phys2bids:
     ------------------------------------------------
-    Filename:            tutorial_file.txt
+    Filename:            tutorial_file_v2.txt
 
     Timepoints expected: 158
-    Timepoints found:    158
+    Timepoints found:    157
     Sampling Frequency:  1000.0 Hz
-    Sampling started at: 0.24499999999989086 s
+    Sampling started at: 0.2459999999998672 s
```

Comment:
@RayStick: I'm sorry for not having caught it earlier, but if it's possible, I would modify a trigger in the middle (e.g. the second) rather than at the beginning: remember that `phy2bids` corrects for missing timepoints with the rationale that they were at the beginning of the recording, but not recorded. For this reason the two examples seem to be almost the same!

This reminds me that in the former tutorial this feature was reported (former line #168). Now it's not there any more, can we add it somewhere, like here?

Reply:
That's strange, on my local copy it says it but here it doesn't. Anyway after "The output" it should have said: 

```
    ------------------------------------------------
    File tutorial_file_v2.txt contains:
    01. Trigger; sampled at 1000.0 Hz
    02. CO2; sampled at 1000.0 Hz
    03. O2; sampled at 1000.0 Hz
    04. Pulse; sampled at 1000.0 Hz
    ------------------------------------------------

    INFO:phys2bids.physio_obj:Counting trigger points
    INFO:phys2bids.physio_obj:The number of timepoints according to the std_thr method is 157. The computed threshold is 1.1506977594943586
    INFO:phys2bids.physio_obj:Checking number of timepoints
    WARNING:phys2bids.physio_obj:Found 1 timepoints less than expected!
    WARNING:phys2bids.physio_obj:Correcting time offset, assuming missing timepoints are at the beginning (try again with a more conservative thr)
    INFO:phys2bids.phys2bids:Plot trigger
    INFO:phys2bids.phys2bids:Preparing 1 output files.
    INFO:phys2bids.phys2bids:Exporting files for freq 1000.0
    INFO:phys2bids.phys2bids:
    ------------------------------------------------
    Filename:            tutorial_file_v2.txt

    Timepoints expected: 158
    Timepoints found:    157
    Sampling Frequency:  1000.0 Hz
    Sampling started at: 0.2459999999998672 s
    Tip: Time 0 is the time of first trigger
    ------------------------------------------------
```

Reply:
I can still change a trigger in the middle though, if you want @smoia ?

Reply:
It would be better, yes, please!

Reply:
Okay, done! 

@vinferrer you have a PR to your branch. 

I changed the tutorial_file_v2.txt so that the trigger is now missing in the middle, not at the start. I then changed the howto doc to match this change, and as well as the png output.

Reply:
PR merged